src/components/MessagePopup/MessagePopup.js

```diff
@@ -0,0 +1,37 @@
+import './MessagePopup.css';
+import Popup from '../Popup/Popup';
+import { usePopups } from '../../contexts/PopupsContext';
+
+function MessagePopup({ name }) {
+  const { isMessagePopupOpen, closeMessagePopup } =
+    usePopups().popups.messagePopup;
+  const { openSignInPopup } = usePopups().popups.signInPopup;
+
+  function handleSignInPopup() {
+    closeMessagePopup();
+    openSignInPopup();
+  }
+
+  return (
+    <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component